src/a08-software-data-integrity-failures/README.md

```diff
@@ -6,8 +6,10 @@ Run the automated tests for verifying the exercise:
 
 `npm run verify`
 
-Start the server for manual tests:
+Start the server and frontend for manual tests:
```

Comment:
<img width="1920" alt="Screenshot 2023-10-16 at 12 42 13" src="https://github.com/nearform/owasp-top-ten-workshop/assets/36735501/3b5ce23f-c426-496f-8aac-496ec324df09">
title is not in the right position and the error returned is not the right one, someone else is causing it not the vulnerability